From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Mon, 11 Nov 2024 13:36:23 +0000
Subject: rm tests-output

Patch to add removal of tests-output/ to clean target in Makefile.am,
in order to avoid a FTBFS on rebuild. This precisely follows the
example in the automake info pages.

See https://bugs.debian.org/1048672
---
 Makefile.am | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index e6b9fe1..d0955fa 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -253,3 +253,6 @@ tests_valgrind_potential_memleaks_SOURCES= tests/valgrind/potential-memleaks.c
 # won't build the BUILT_SOURCES.
 test: all
 	$(top_srcdir)/tests/test_scrypt.sh .
+
+clean-local:
+	-rm -rf tests-output
